src/infrastructure/rendering/table_image_renderer.py

```
@@ -146,13 +234,211 @@ def render_table(self, table_html: str | Tag) -> TableImageRenderResult | None:
+
+        if removed > 0:
+            logger.debug("Table image cache GC removed %s files", removed)
+        return removed
+
+    def _run_periodic_cache_gc(
+        self,
+        *,
+        now_ts: float,
+        skip_paths: set[Path] | None = None,
+    ) -> int:
+        """按配置间隔触发表格图 GC，避免每次渲染都全量扫描 cache。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)